### PR TITLE
Add TLS/Auth via exporter-toolkit web-config

### PR DESCRIPTION
**Pull request by michaelpsomiadis**
Hi, thanks for this awesome exporter!

In our environment we usually run TLS with auth and I noticed it was missing on this exporter but found that it wasn't too hard to add. With this patch I can now parse my existing [web-config.yml](https://github.com/prometheus/exporter-toolkit/blob/master/docs/web-configuration.md) file and it enables the use of TLS and auth nicely.

I'm nowhere near a golang dev, I built my test versions with go1.18.1 which seems lots newer since the last release.
The only issue I'm facing is no matter what version I build with it seems to want to print all debug information even when I specify `--log.level=info` or even `--log.level=warn`. Maybe it's something I'm doing wrong.

Cheers.

### CODE REVIEW

**Comment by gvengel** [comment]
Hey @michaelpsomiadis!

Thanks for the kind words. I'll find some time to review this soon and keep you updated.

-Gabe

File: main.go [hunk 1]
```diff
@@ -37,6 +38,7 @@ var (
 	useJournal       = kingpin.Flag("exim.use-journal", "Use the journal instead of log file tailing").Envar("EXIM_USE_JOURNAL").Bool()
 	syslogIdentifier = kingpin.Flag("exim.syslog-identifier", "Syslog identifier used by Exim").Default("exim").Envar("EXIM_SYSLOG_IDENTIFIER").String()
 	tailPoll         = kingpin.Flag("tail.poll", "Poll logs for changes instead of using inotify.").Envar("TAIL_POLL").Bool()
+	configFile       = kingpin.Flag("web.config", "Path to config yaml file that can enable TLS or authentication.").Default("").String()
```
Comment (gvengel) [inline main.go:41]: I haven't used the exporter-toolkit before. Should this be `web.config.file` per [flag.go](https://github.com/prometheus/exporter-toolkit/blob/master/web/kingpinflag/flag.go#L23)?

A number of users run the exporter under Docker or Kubernetes and can't control the command line arguments directly. We'll need to add `.Envar("WEB_CONFIG_FILE")` to support them.

**Comment by gvengel** [comment]
Looks good overall. 

Yeah, I've been meaning to update all dependencies and do a new build against golang 1.18. Once we get this feature hammered out, I'll do that before cutting a new release.

I haven't run into the issue you describe with `--log.level`. Once I've updated everything to 1.18, I'll do some testing an see if I can reproduce the problem.

**Comment by gvengel** [comment]
Ah, I'm actually able to reproduce the log level issue. I'll work on a fix tomorrow. 🍻

File: main.go (hunk 1 above)
Comment (michaelpsomiadis) [reply main.go]: Ah, I was using [node_exporter](https://github.com/prometheus/node_exporter/blob/master/node_exporter.go#L165) as an example but `web.config.file` works. I remember seeing both flags around in other exporters maybe the name was updated at some point. 🤷

I've updated the flag name and added the Envar, tested it locally and both seem to work :)

**Comment by gvengel** [comment]
This has been released in version 1.4.0! Thanks for contributing! 🍻

**Pull request merged**